### ISSUE TITLE
pixel_cls_logits_flatten and pos_mask have different dimention while calculating tf.nn.sparse_softmax_cross_entroy_with_logits

### ISSUE THREAD

**Issue by taruntechnolofies** [ZJULearning/pixel_link#97]
in pixel_link_symbol.py

pixel_cls_loss = tf.nn.sparse_softmax_cross_entropy_with_logits(
                    logits = self.pixel_cls_logits_flatten, 
                    labels = tf.cast(pos_mask, dtype = tf.int32))

logits and labels have different dimentions while calculating loss.
logits dim is [BATCH_SIZE, 16384, 2]
labels dim is [BATCH_SIZE, 16384]
why is it different? what is is effect on pixel_cls_loss ?how is the loss calculates then?

**Comment by wulouzhu** [comment]
I have the same problem
